src/components/gltf-model-plus.js

```diff
@@ -274,7 +369,13 @@ AFRAME.registerComponent("gltf-model-plus", {
       }
 
       const gltfPath = THREE.LoaderUtils.extractUrlBase(src);
-      const model = await cachedLoadGLTF(src, this.data.basePath, this.data.contentType, this.preferredTechnique);
+
+      if (!GLTFCache[src]) {
+        GLTFCache[src] = loadGLTF(src, this.data.basePath, this.data.contentType, this.preferredTechnique);
+      }
+
+      const cachedModel = await GLTFCache[src];
+      const model = cloneGltf(cachedModel);
```

Comment:
This will still keep the broken promise in the `GLTFCache`. You need to evict it from the cache if it fails.

Reply:
Oh, right.